represetnation_labels/plot_functions.py

```diff
@@ -0,0 +1,918 @@
+    pal_r = Panel(child=reletive, title='Compare Populations')
+    tabs = Tabs(tabs = [pal_nr,pal_r])
+    if out:
+        file_name = str(date.today()) + 'tabs_plot.html'
+        output_file(file_name)
+        save(tabs)
+    return(tabs)
+
+def all_datasets(plot_dict,out):
+    panal_list = [Panel(child = rel_plots(plot_dict,i,out= False),title = i) for i in plot_dict.keys()]
+    tabs = Tabs(tabs = panal_list)
+    title = Div(text="""Data Representation Labels""",
+                style={'font-size': '28pt', 'color': '#a0a0a0', 'font': 'helvetica'}
+                )
+    created_by = Div(
+        text="""<i>Created by Wellcome’s Data for Science and Health Priority Area </i>""",
+        style={'font-size': '7pt', 'color': '#555555', 'font': 'helvetica'}
+    )
+    description  = Div(
+        text="""This is a tool for researchers to examine the demographics of commonly used datasets and how they compare to the UK population. The datasets we are highlighting are: """,
+        style={'font-size': '14pt', 'color': '#555555', 'font': 'helvetica'}
+    )
+
+    dataset_list = Div(
+        text="""<ul><li>UK Biobank</li><li>ALSPAC</li><li>CPRD</li><li>National Child Development Study</li><li>Whitehall study II</li><li>HES</li></ul>  """,
+        style={'font-size': '14pt', 'color': '#555555', 'font': 'helvetica'})
+
+    creation = Div(
+        text="""It was created by the Wellcome Trust with the intention of comparing how the UK population is represented in these datasets, and highlighting where there are disparities.<br>
+         <b>How we chose the datasets and accessed the number going into the graph:</b>  The datasets represented by these labels are some of the most commonly used and cited datasets in the UK today. The data displayed here was collated using a combination of metadata available in published papers and online platforms (such as Closer Discovery and the datasets own webpages). No raw data was accessed for the purpose of this project.<br>
+         <b>Known limitations:</b> We know that the groupings of sub-populations used in the datasets, e.g. ethnicity groupings, are subjective and potentially inaccurate at times.<br>
+         Please don’t hesitate to contact us with any questions, feedback or suggestions at <u>b.knowles@wellcome.org</ul> """,
+        style={'font-size': '14pt', 'color': '#555555', 'font': 'helvetica'}
+    )
+
+    last_updated = Div(
+        text="""<i>Date last updated: 9th Nov 2020</i> """,
+        style={'font-size': '7pt', 'color': '#555555', 'font': 'helvetica'}
+
+    )
+    final = gridplot([[title],[created_by],[description], [dataset_list],[creation],[last_updated] ,[tabs]],
+                     toolbar_options={'autohide': True})
+
+    if out:
+        file_name = str(date.today()) + 'representation_labels.html'
+        output_file(file_name)
+        save(final)
+    return(final)
+
+
+
+if __name__ == '__main__':
+    all_datasets(graph_dict2, out=True)
```

Comment:
`NameError: name 'graph_dict2' is not defined`. I think you need the stuff you had in useful_functions here instead